config/locales/diaspora/en.yml

```diff
@@ -427,9 +427,9 @@ en:
       restrict_posts_i_see_q: "Can I restrict the posts in my stream to just those from certain aspects?"
       restrict_posts_i_see_a: "Yes. Click on “My aspects” in the side-bar and then click individual aspects in the list to select or deselect them. Only the posts by people in the selected aspects will appear in your stream."
       rename_aspect_q: "Can I rename an aspect?"
-      rename_aspect_a: "Yes. In your list of aspects on the left side of the main page, point your mouse at the aspect you want to rename. Click the little “Edit” pencil that appears to the right. You will be taken to the page for this aspect. Click the edit pencil next to the aspect name at the top of this page, the change the name and press “Update”."
+      rename_aspect_a: "Yes. Go to the “Contacts” page form the user menu and select the aspect you want to rename. You can also go there from the stream : click on “My aspects” on the left side of the main page, point your mouse at the aspect you want to rename and click the little “Edit” pencil that appears to the right. You will be taken to the page for this aspect. Click the edit pencil next to the aspect name at the top of this page, then change the name and press “Update”."
```

Comment:
This answer is presented like this because previously this pencil in the sidebar was the only way to open a facebox allowing to rename the aspect. It was the only way to rename an aspect. Now the classic way would be to open the contacts page and select the aspect and then rename it. The pencil still exists but is just a shortcut that's why I'm wondering if we should talk about it.